src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor().toString();
+        int number = getRandomNumber();
+        Ball ball = new Ball();
+        ball.setColor(color);
+        ball.setNumber(number);
+        return ball;
+    }
+
+    public int getRandomNumber() {
+        return new Random().nextInt(100);

```

Comment:
Think about which variables should be local in the method and which should be class-level